### PR TITLE
Fix ping in game titles

### PR DESCRIPTION
**Pull request by asquared31415**
Makes pings not actually ping if present in a game title by using a non breaking space.

### CODE REVIEW

File: GamesCommand.cs [hunk 1]
```diff
@@ -33,8 +33,9 @@ public async Task MessageReceivedAsync(SocketMessage message)
         {
             if (message.Content.StartsWith("!addgame "))
             {
-                var game = message.Content.Substring("!addgame ".Length).ToLower();
-                await AddGame(message, game);
+	            // For some reason the Discord library made the mention sanitizer internal :c but this nbsp is what they do
+	            var game = message.Content.Substring("!addgame ".Length).ToLower().Replace("@", "@\u200B");
+	            await AddGame(message, game);
```
Comment (khyperia) [inline GamesCommand.cs]: can you fix the mixing of tabs and spaces here

Comment (asquared31415) [reply GamesCommand.cs]: Sorry about that, I tried to check before but my IDE must have done something weird.

**Comment by asquared31415** [comment]
I also just realized while fixing spacing that delgame needs to have the same replacement, so I added that too.

**Comment by khyperia** [comment]
Ah, sorry, I just realized, this breaks the current database - the _output message_ needs to be sanitized, not the value that goes into the database. Otherwise, the current DB will be broken. (I also just don't like the idea of storing santitized data in the DB as it screws with versioning, it's all kinds of gross, formatting messages is the proper place for that code)

**Comment by asquared31415** [comment]
Ah okay I will move that to output, that does make more sense, idk why I didn't think about that at the time.

**Pull request merged**